packages/synapse-react-client/src/components/Markdown/MarkdownPopover.tsx

```diff
@@ -37,6 +37,9 @@ const buttonBoxSx = {
   },
 }
 
+// Register a global atom to track which popover is open, to ensure only one is shown at any given time
+const openMarkdownPopoverAtom = atom<string | null>(null)
```

Comment:
This is how our bootstrap 3 "super tooltip" works, so let's match that behavior here.